package.json

```
@@ -34,7 +34,9 @@
     "test": "cd packages/server && yarn jest",
     "lint": "eslint --ext \".js,.ts,.tsx\" --ignore-path .gitignore .",
     "lint-fix": "yarn lint --fix && manypkg fix",
-    "postinstall": "preconstruct dev",
+    "codegen": "yarn ts-node scripts/generate-big-f-router.ts",
+    "ts-node": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\"",
```

ts-node options might be able to go in tsconfig these days. IMO transpileOnly should be turned on too, since there are already CI checks for type correctness. Failing that, I think there could be one fewer escape with single quotes
```suggestion
    "ts-node": "ts-node --compiler-options '{\"module\":\"commonjs\"}'",
```